### PR TITLE
Update Set-DlpCompliancePolicy.md

### PR DESCRIPTION
**Pull request by ybhargav1995**

### CODE REVIEW

File: exchange/exchange-ps/exchange/Set-DlpCompliancePolicy.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ Accept wildcard characters: False
 ```
 
 ### -RetryDistribution
-The RetryDistribution switch redistributes the policy to all OneDrive for Business and SharePoint Online locations. You don't need to specify a value with this switch.
+The RetryDistribution switch redistributes the policy to all Excahnge,OneDrive for Business and SharePoint Online locations. You don't need to specify a value with this switch.
```
Comment (JohanFreelancer9) [inline exchange/exchange-ps/exchange/Set-DlpCompliancePolicy.md:132]:
```suggestion
The RetryDistribution switch redistributes the policy to all Exchange, OneDrive for Business, and SharePoint Online locations. You don't need to specify a value with this switch.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ybhargav1995, kindly commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@dariomws @get-itips @tseward

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by dariomws** [comment]
If you have a Microsoft alias, you can link your accounts in GitHub so that a blue "Microsoft submitter" label appears to any PR's or issue you submit. Guidance on it here: https://docs.opensource.microsoft.com/tools/github/index.html

Here is the site you can use to see the mappings between GitHub and Microsoft aliases: https://repos.opensource.microsoft.com/people?q=ybhargav1995

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thank

**Comment by chrisda** [comment]
Issue #9926

**Pull request merged**